scheduler/src/cook/mesos/pool.clj

```diff
@@ -70,7 +71,7 @@
   (let [pools (all-pools db)
         default-pool-name (config/default-pool)]
     (log/info "Pools in the database:" pools ", default pool:" default-pool-name)
-    (if default-pool-name
```

Comment:
Would this be better in the config section, so everyone else doesn't have to worry about it?

Reply:
By "this", do you meany the `blank?` check?

Reply:
Right, replacing blank with nil. Sorry for the lack of clarity.

Reply:
Done